lib/pbench/cli/agent/commands/results/move.py

```diff
@@ -97,7 +97,6 @@ def execute(self, single_threaded: bool, delete: bool = True) -> int:
 
                 try:
                     crt = CopyResultTb(
-                        self.context.controller,
```

Comment:
Are we sure we want to remove that? We are providing controller or anyway that is just going to be added in metadata file.

Reply:
I _think_ all that matters (in this context) is that the Agent can find where the results are.  Apparently, they can be in a non-standard location which the user can point us to using the `controller` option.  If we can find them using that option, then we really don't care (as far as I know) what the value was that the user specified.  (That is, for the moment, there is a requirement that the value be a legal directory name, but that's much less restrictive than requiring it to be a legal host name, which is what the current check does, and we can let the filesystem itself decide whether the value is appropriate -- we don't have to.  Moreover, soon, hopefully, we won't even need it to be a legal directory name.)

So, I don't think we need to validate the value in any way other than to try to use it -- if it works, then everone is happy.  We just need to make sure that we gracefully handle any errors which result from it not working.

Reply:
In terms of the *agent*, the "controller" is the hostname on which the collection utilities are running when the result directory tree and tarballs are formed. But really it's just an identifier... and on the server side it's just a string.

Right now we're only getting here from the agent CLI which primes this with `hostname`; but when we generalize this as a callable API, why would we care whether the caller is using `hostname` or some other (likely much more useful) identifier? (Especially in a cloud environment where hostname is probably pretty much meaningless.)

On the other hand, I think this is a more strategic/architectural decision than just "oh I'm touching this area of code so I might as well delete the check" and probably doesn't belong in this PR.

Reply:
>  I think this is a more strategic/architectural decision than just "oh I'm touching this area of code so I might as well delete the check" and probably doesn't belong in this PR.

I don't care strongly, but I remember being bitten by this check (or one like it) the last time we tried to do bulk uploads of existing results...and I'd rather not be bitten by it again, especially since I think we all agree that the check is overzealous.  And, I can't really say whether it belongs in this PR, but it certainly fits under the _title_ of this PR.  😉